server/Routers/exploreRouter.js

```diff
@@ -5,8 +5,13 @@ const router = express.Router();
 
 // get router for explore page
 router.get('/', ideaController.getIdeas, (req, res) => {
-  // console.log('res.locals.ideas', res.locals.ideas);
+  console.log('res.locals.ideas', res.locals.ideas);
```

Comment:
Best practice not to include `console.log`s. Please try to comb through and remove them before PR'ing in the future.